spec/requests/keywords_controller_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe KeywordsController, type: :request do
+  describe 'GET #index' do
+    it 'returns http success' do
+      sign_in Fabricate(:user)
+      get :index
+      expect(response).to have_http_status(:success)
```

Comment:
We usually keep the assert matching stuff to a separate line as a group. To make it more readable the tests
```suggestion

      expect(response).to have_http_status(:success)
```

Reply:
Updated in [`067c022` (#37)](https://github.com/suho/google-search-ruby/pull/37/commits/067c0225bed0a50f9947e8747b64a6b210c6e73d)